node_scenarios.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import time
+import json
+import boto3
+import random
+import logging
+import kraken.kubernetes.client as kubecli
+import kraken.invoke.command as runcommand
+
+
+def node_start_stop_scenario(node_scenario_config):
+    selector = node_scenario_config["selector"]
+    number_of_kills = node_scenario_config["number_of_kills"]
+
+    client = boto3.client('ec2')
+    instance = boto3.resource('ec2').Instance('id')
+
+    for _ in range(number_of_kills):
+        try:
+            logging.error("Starting node_start_stop_scenario injection")
+            nodes = kubecli.list_nodes(label_selector=selector)
+            if not nodes: raise Exception("Nodes with the provided selector do not exist")
+            logging.info("Nodes with the selector %s: %s" % (selector, nodes))
+            number_of_nodes = len(nodes)
+            node = nodes[random.randint(0, number_of_nodes - 1)]
+            instance_id = client.describe_instances(Filters=[{'Name': 'private-dns-name', 'Values': [node]}])['Reservations'][0]['Instances'][0]['InstanceId']
+            logging.info("Stopping the node %s with instance ID: %s " % (node, instance_id))
+            client.stop_instances(InstanceIds=[instance_id])
+            instance.wait_until_stopped(InstanceIds=[instance_id])
+            logging.info("Node with instance ID: %s is in stopped state" % (instance_id))
+            time.sleep(30)
```

Comment:
Let's move the time out to config after changing this to poll instead of sleep. This way the user can set the time out after which the test can fail in case the node is not in the expected state.